Makefile

```diff
@@ -16,7 +16,7 @@ benchmark:
 
 # --- Below only needed for releasing new versions
 
-LIB_PG_QUERY_TAG = 16-5.0.0
+LIB_PG_QUERY_TAG = windows-support
```

Comment:
This is temporary and will be adjusted once we have a tagged libpg_query release (and before merging).